tests/lib/cli.js

```diff
@@ -633,6 +633,30 @@ describe("cli", function() {
         });
     });
 
+    describe("when passed --no-inline-config", function() {
+
+        it("should pass allowInlineConfig:false to CLIEngine and ignore inline config when executing on text", function() {
```

Comment:
Oops. I think I misunderstand the test then.

It's passing `--no-inline-config` which sets `allowInlineConfig` to `false`, and then the linting exists with a non-zero because it finds a rule violation that would normally be ignored as it's on a `// eslint-disable-line` line, but the `allowInlineConfig = false` means that comment config is ignored.